cmake/Corrosion.cmake

```diff
@@ -24,6 +24,10 @@ get_property(
     TARGET Rust::Cargo PROPERTY IMPORTED_LOCATION
 )
 
+if (CMAKE_VERSION VERSION_GREATER_EQUAL 3.19.0)
```

Comment:
Since this is a rather large change, i would like to have CMake option to toggle this off in addition to the version check.

Reply:
done